### PR TITLE
[Keras] add BatchNorm data gamma argument

### PR DESCRIPTION
**Pull request by yajiedesign**
fix symbol.py __getitem__ error

### CODE REVIEW

File: src/operator/batch_norm-inl.h [hunk 1]
```diff
@@ -32,6 +32,7 @@ struct BatchNormParam : public dmlc::Parameter<BatchNormParam> {
   float momentum;
   bool fix_gamma;
   bool use_global_stats;
+  bool full_output;
```
Comment (piiswrong) [inline src/operator/batch_norm-inl.h]: bool output_mean_var
